### DISCUSSION TITLE
compile Webots from source,can not lancher webots

### DISCUSSION THREAD

**Discussion by DOMINICQQQ** [cyberbotics/webots#4511] [Q&A]
<img width="443" alt="3" src="https://user-images.githubusercontent.com/100404516/166878169-267958fa-4a2d-475d-bde8-504c094fdcd5.png">

**Comment by omichel** [comment]
Did you follow the instructions at https://github.com/cyberbotics/webots/wiki/Windows-installation?
It shouldn't display the error about `bash_profile.windows` not found.
